examples/lessons/10/app.js

```diff
@@ -0,0 +1,157 @@
+/* eslint-disable no-var, max-statements */
+/* eslint-disable array-bracket-spacing, no-multi-spaces */
+/* global document */
+import {
+  GL, AnimationLoop, loadTextures, loadFile, addEvents,
+  resetParameters, setParameters
+} from 'luma.gl';
+
+import {
+  Matrix4
+} from 'math.gl'
+
+import {loadWorldGeometry, World} from './world';
+
+var pitch = 0;
+var pitchRate = 0;
+var yaw = 0;
+var yawRate = 0;
+var xPos = 0;
+var yPos = 0.4;
+var zPos = 0;
+var speed = 0;
+var direction = [0, 0, -1];
+
+const animationLoop = new AnimationLoop({
+  onInitialize: ({canvas, gl}) => {
+    addKeyboardHandler(canvas);
+
+    setParameters(gl, {
+      clearColor: [0, 0, 0, 1],
+      clearDepth: 1,
+      depthTest: true
+    });
+
+    return loadTextures(gl, {
+      urls: ['mud.gif']
+    })
+    .then(textures => {
+		  return loadFile({url: 'world.txt'})
+		  .then(file => {
+        var geometry = loadWorldGeometry(file);
+        var world = new World({
+        gl,
+        texture: textures[0],
+        geometry});
+        return {world};
+		  });
+    });
+  },
+  onRender: ({
+    gl, tick, aspect, world
+  }) => {
+    // Update Camera Position
+    const eyePos = [xPos, yPos, zPos];
+    const centerPos = new Matrix4()
+      .identity()
+      .rotateX(degToRad(pitch))
+      .rotateY(degToRad(yaw))
+      .transformVector3(direction)
+      .add(eyePos);
+    gl.clear(GL.COLOR_BUFFER_BIT | GL.DEPTH_BUFFER_BIT);
+
+    var uMVMatrix = new Matrix4()
+      .lookAt({eye: eyePos, center: centerPos, up:[0, 1, 0]})
+
+    world.render({
+      uMVMatrix,
+      uPMatrix: new Matrix4().perspective({fov: 45 * Math.PI / 180, aspect, near: 0.1, far: 100})
+    });
+    handleKeys();
+    animate();
+  }
+});
+
+animationLoop.getInfo = () => {
+  return `
+  <p>
+    <a href="http://learningwebgl.com/blog/?p=1008" target="_blank">
```

Comment:
This text and link should be updated?

Reply:
This is intended I think cause in luma.gl's example web page we need show the link to webgl lesson pages.

Reply:
Done.